examples/mobile_tests/src/lib.rs

```diff
@@ -96,41 +91,6 @@ fn test_get_by_id_and_sign() {
     let key = SecureEnvironment::get_keypair_by_id(id).unwrap();
     let msg  = b"Hello World!";
 
-    let signature = key.sign(msg).unwrap();
-
-
-    assert_eq!(signature.len(), 64);
-}
-
-fn test_generate_and_sign_and_verify_with_askar() {
-    let id = Uuid::new_v4();
-    let key = SecureEnvironment::generate_keypair(id).unwrap();
-    let public_key = key.get_public_key().unwrap();
-    let msg  = b"Hello World!";
-
-    let signature = key.sign(msg).unwrap();
-
-    let keypair = P256KeyPair::from_public_bytes(&public_key).unwrap();
-
-    let is_valid = keypair.verify_signature(msg, &signature);
-
-    assert!(is_valid);
-
-}
-
-fn test_get_by_id_and_sign_and_verify_with_askar() {
-    let id = Uuid::new_v4();
-    SecureEnvironment::generate_keypair(id).unwrap();
-    let key = SecureEnvironment::get_keypair_by_id(id).unwrap();
-    let public_key = key.get_public_key().unwrap();
-    let msg  = b"Hello World!";
-
-    let signature = key.sign(msg).unwrap();
-
-    let keypair = P256KeyPair::from_public_bytes(&public_key).unwrap();
-
-    let is_valid = keypair.verify_signature(msg, &signature);
-
-    assert!(is_valid);
```

Comment:
I think we should keep an option to lower the security settings so we can have a test that does signing + verification. As now we basically have now tests for this.

Reply:
And we should probably also have some negative tests. Wrong signature / changed content should give an error

Reply:
> I think we should keep an option to lower the security settings so we can have a test that does signing + verification. As now we basically have now tests for this. we decided not to make it optional, right? Or just in askar?

Reply:
> And we should probably also have some negative tests. Wrong signature / changed content should give an error

this library does not validate the signature, thats from askar.